s2and/model.py

```diff
@@ -550,7 +570,9 @@ def predict_incremental(self, block_signatures: List[str], dataset: ANDData):
                     cluster_seeds_require_inverse[cluster_num].append(signature_id)
                 for altered_cluster_num in altered_cluster_nums:
                     signature_ids_for_cluster_num = cluster_seeds_require_inverse[altered_cluster_num]
-                    reclustered_output, _ = self.predict({"block": signature_ids_for_cluster_num}, dataset)
+                    reclustered_output, _ = self.predict(
+                        {"block": signature_ids_for_cluster_num}, dataset, incremental_dont_use_cluster_seeds=True
```

Comment:
This is hardcoded to true? Can you explain please?

Reply:
this is the "split" operation for splitting up claimed profiles (e.g. Iz vs Islam) to be compatible with what s2and originally thought. when we call predict on this line of code, we never want to use the original cluster seeds, because they contain that incompatibility and our goal is to revert to what s2and would've done without seeds.

Reply:
the arg defaults to False everywhere else.

Reply:
You might want to put this logic in the doc somewhere?